apps/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 add_executable(m17-demod m17-demod.cpp)
-target_link_libraries(m17-demod PRIVATE m17cxx ${CODEC2_LIBRARIES} ${Boost_LIBRARIES})
+target_link_libraries(m17-demod PRIVATE m17cxx codec2 Boost::program_options)
```

Comment:
when linking codec2 like this, it should probably be prefixed (similar to boost) to signify that an external library is used.

Reply:
Here's what Ryan wrote about that change, does it make sense to you?  If not - I'll go back and find out more info.

```
Subject: [PATCH] Switch to use find_package for codec2, link with codec2
 target.

Codec2 breaks standard and calls its import library "libcodec2.lib",
which doesn't work with the Unix-style pkg-config file that tells
programs to link with "codec2"=="codec2.lib". Codec2 has been providing
its own codec2-config.cmake file for a while now, so just use it to make
things simpler.

```
https://github.com/conda-forge/m17-cxx-demod-feedstock/blob/main/recipe/0009-Switch-to-use-find_package-for-codec2-link-with-code.patch